stoservice/src/main/java/com/tutrit/stoservice/service/CustomerService.java

```diff
@@ -0,0 +1,51 @@
+package com.tutrit.stoservice.service;
+
+import com.tutrit.stoservice.bean.Customer;
+import com.tutrit.stoservice.context.ApplicationContext;
+import com.tutrit.stoservice.repository.CustomerRepository;
+import com.tutrit.stoservice.utility.ParseCommand;
+
+import java.util.*;
+
+public class CustomerService {
+
+    public static Map<String, String> parseBody() {
+        List<Integer> positions = new ArrayList<>();
+        String body = ApplicationContext.get(ParseCommand.class).getBody();
```

Comment:
I don't think it's necessary to take ParseCommand.class out of context. Methods of ParseCommand.class can be made static